consensus/tendermint/timeout.go

```diff
@@ -1,5 +1,16 @@
 package tendermint
 
+type timeout struct {
+	Step   step     `cbor:"s"`
+	Height height   `cbor:"h"`
+	Round  round    `cbor:"r"`
```

Comment:
I believe this `cbor` tags will be ignored since we have the `toarray`. I think it's safe to remove them